pkg/resourceoverrides/compute_overrides.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package resourceoverrides
+
+import (
+	"fmt"
+
+	"github.com/GoogleCloudPlatform/k8s-config-connector/pkg/k8s"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	apiextensions "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+func GetComputeMangedSSLCertificateResourceOverrides() ResourceOverrides {
+	return ResourceOverrides{
+		Kind: "ComputeManagedSSLCertificate",
+		Overrides: []ResourceOverride{
+			mapCertificateIdToCertificateID(),
+		},
+	}
+}
+
+func mapCertificateIdToCertificateID() ResourceOverride {
+	return ResourceOverride{
+		CRDDecorate: func(crd *apiextensions.CustomResourceDefinition) error {
```

Comment:
For my own eduction, since the CRD field is already renamed to "certificateID" (as the result of the patch in TF), why do we still need CRDDecorate override?